### PR TITLE
OpenZWave 0.5.2

### LINKED ISSUE #1454: ozwave 0.5.0 crashing
ozwave is crashing for me after updating it from 0.4 to 0.5

```[20200710 16:45:21.032 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/refreshvalue/") 
[20200710 16:45:21.032 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "removeassociation" 
[20200710 16:45:21.032 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/removeassociation/") 
[20200710 16:45:21.032 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "removefailednode" 
[20200710 16:45:21.032 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/removefailednode/") 
[20200710 16:45:21.032 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "removenode" 
[20200710 16:45:21.032 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/removenode/") 
[20200710 16:45:21.032 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "replacefailednode" 
[20200710 16:45:21.032 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/replacefailednode/") 
[20200710 16:45:21.032 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "requestallconfigparam" 
[20200710 16:45:21.032 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/requestallconfigparam/") 
[20200710 16:45:21.032 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "requestconfigparam" 
[20200710 16:45:21.032 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/requestconfigparam/") 
[20200710 16:45:21.032 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "requestnetworkupdate" 
[20200710 16:45:21.033 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/requestnetworkupdate/") 
[20200710 16:45:21.033 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "requestnodedynamic" 
[20200710 16:45:21.033 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/requestnodedynamic/") 
[20200710 16:45:21.033 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "requestnodeneighborupdate" 
[20200710 16:45:21.033 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/requestnodeneighborupdate/") 
[20200710 16:45:21.033 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "requestnodestate" 
[20200710 16:45:21.033 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/requestnodestate/") 
[20200710 16:45:21.033 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "sendnodeinformation" 
[20200710 16:45:21.033 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/sendnodeinformation/") 
[20200710 16:45:21.033 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "setpollinterval" 
[20200710 16:45:21.033 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/setpollinterval/") 
[20200710 16:45:21.033 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "setvalue" 
[20200710 16:45:21.041 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/setvalue/") 
[20200710 16:45:21.041 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "softresetcontroller" 
[20200710 16:45:21.042 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/softresetcontroller/") 
[20200710 16:45:21.042 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "syncronisenodeneighbors" 
[20200710 16:45:21.042 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/syncronisenodeneighbors/") 
[20200710 16:45:21.042 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "testnetwork" 
[20200710 16:45:21.042 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/testnetwork/") 
[20200710 16:45:21.042 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "testnetworknode" 
[20200710 16:45:21.042 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/testnetworknode/") 
[20200710 16:45:21.045 CEST] [ozw.library] [info]: Info - Node: 0 Lookup for mfs.db.openzwave.com returned 108 
[20200710 16:45:21.085 CEST] [ozw.mqtt.publisher] [debug]: Topics:  "OpenZWave/1/status/" 
[20200710 16:45:21.086 CEST] [ozw.mqtt.publisher] [warning]: Another ozwdaemon is running for Instance  1 
[20200710 16:45:21.086 CEST] [ozw.mqtt.publisher] [warning]: If not, please clean up the MQTT Topic:  "OpenZWave/1/status/" 
[20200710 16:45:21.086 CEST] [ozw.mqtt.publisher] [warning]: "{\n    \"OpenZWave_Version\": \"1.6.1201\",\n    \"OZWDaemon_Version\": \"0.1.0\",\n    \"QTOpenZWave_Version\": \"1.2.0\",\n    \"QT_Version\": \"5.14.2\",\n    \"Status\": \"driverRemoved\",\n    \"TimeStamp\": 1594392276,\n    \"homeID\": 0\n}" 
[20200710 16:45:21.086 CEST] [ozw.library] [debug]: Detail - Node: 0 Notification: DriverRemoved 
[20200710 16:45:21.086 CEST] [ozw.notifications] [debug]: Notification pvt_driverRemoved  0 Thread:  0x7fb69f4e38 
[20200710 16:45:21.086 CEST] [ozw.mqtt.publisher] [debug]: Publishing Event stopped 
[20200710 16:45:21.086 CEST] [ozw.mqtt.publisher] [debug]: Publishing Event driverRemoved: 0 
[20200710 16:45:21.087 CEST] [ozw.library] [debug]: Always - Node: 0 *************************************************************************** 
[20200710 16:45:21.088 CEST] [ozw.library] [debug]: Always - Node: 0 *********************  Cumulative Network Statistics  ********************* 
[20200710 16:45:21.088 CEST] [ozw.library] [debug]: Always - Node: 0 *** General 
[20200710 16:45:21.088 CEST] [ozw.library] [debug]: Always - Node: 0 Driver run time: . .  . 0 days, 0 hours, 0 minutes 
[20200710 16:45:21.088 CEST] [ozw.library] [debug]: Always - Node: 0 Frames processed: . . . . . . . . . . . . . . . . . . . . 0 
[20200710 16:45:21.088 CEST] [ozw.library] [debug]: Always - Node: 0 Total messages successfully received: . . . . . . . . . . 0 
[20200710 16:45:21.088 CEST] [ozw.library] [debug]: Always - Node: 0 Total Messages successfully sent: . . . . . . . . . . . . 0 
[20200710 16:45:21.088 CEST] [ozw.library] [debug]: Always - Node: 0 ACKs received from controller:  . . . . . . . . . . . . . 0 
[20200710 16:45:21.088 CEST] [ozw.library] [debug]: Always - Node: 0 *** Errors 
[20200710 16:45:21.088 CEST] [ozw.library] [debug]: Always - Node: 0 Unsolicited messages received while waiting for ACK:  . . 0 
[20200710 16:45:21.088 CEST] [ozw.library] [debug]: Always - Node: 0 Reads aborted due to timeouts:  . . . . . . . . . . . . . 0 
[20200710 16:45:21.088 CEST] [ozw.library] [debug]: Always - Node: 0 Bad checksum errors:  . . . . . . . . . . . . . . . . . . 0 
[20200710 16:45:21.089 CEST] [ozw.library] [debug]: Always - Node: 0 CANs received from controller:  . . . . . . . . . . . . . 0 
[20200710 16:45:21.089 CEST] [ozw.library] [debug]: Always - Node: 0 NAKs received from controller:  . . . . . . . . . . . . . 0 
[20200710 16:45:21.090 CEST] [ozw.library] [debug]: Always - Node: 0 Out of frame data flow errors:  . . . . . . . . . . . . . 0 
[20200710 16:45:21.090 CEST] [ozw.library] [debug]: Always - Node: 0 Messages retransmitted: . . . . . . . . . . . . . . . . . 0 
[20200710 16:45:21.090 CEST] [ozw.library] [debug]: Always - Node: 0 Messages dropped and not delivered: . . . . . . . . . . . 0 
[20200710 16:45:21.090 CEST] [ozw.library] [debug]: Always - Node: 0 *************************************************************************** 
[20200710 16:45:21.090 CEST] [ozw.library] [warning]: Warning - Node: 0 WARNING: Tried to write driver config with no home ID set 
[20200710 16:45:21.092 CEST] [ozw.library] [info]: Info - Node: 0 Stopping DNSThread 
[16:45:21] INFO: Starting ozw-admin...
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
ozwadmin: Settings Path is at  "/root/.config/OpenZWave/ozw-admin.conf"
ozwadmin: Checking  "/data/ozw/config"  for manufacturer_specific.xml
ozwadmin: Checking  "/data/ozw/config"  for options.xml
ozwadmin: DBPath:  "/data/ozw/config/"
ozwadmin: userPath:  "/data/ozw/config/"
WebSocket server settings:
  - Listen on :8099
  - Web server. Web root: /usr/share/novnc
  - No SSL/TLS support (no cert file)
  - proxying from :8099 to 127.0.0.1:5900
ozwadmin: Settings Path:  "/root/.config/OpenZWave/ozw-admin.conf"
ozwadmin: Using Saved Network Cache Size: 1000
QT_MESSAGE_PATTERN: %{backtrace} is not supported by this Qt build
[20200710 16:45:22.078 CEST] [ozwadmin] [info]: OZWCore Initilized QTOpenZwave(0x559b6ec100) QTOZWManager(0x559b72fb80) 
[20200710 16:45:22.133 CEST] [default] [warning]: QVncServer created on port 5900 
[20200710 16:45:22.296 CEST] [ozw.library] [debug]: Debug - Node: 0 Downloads Remaining: 0 
[20200710 16:45:22.296 CEST] [ozw.library] [info]: Info - Node: 0 ManufacturerSpecificDB Initialized 
[20200710 16:45:24.302 CEST] [ozw.library] [info]: Info - Node: 0 mgr,     Driver for controller /dev/serial/by-id/usb-0658_0200-if00 removed 
[20200710 16:45:24.302 CEST] [ozw.manager] [debug]: Driver Removed for  "/dev/serial/by-id/usb-0658_0200-if00" 
[20200710 16:45:24.302 CEST] [ozw.manager] [debug]: OZW Serial Port Closed 
[20200710 16:45:24.302 CEST] [ozw.daemon] [info]: Shutting Down  "ozwdaemon"  Version:  "0.1.0" 
[20200710 16:45:24.303 CEST] [ozw.daemon] [info]: OpenZWave Version:  "1.6.1201" 
[20200710 16:45:24.303 CEST] [ozw.daemon] [info]: QT-OpenZWave Version:  "1.2.0" 
[20200710 16:45:24.303 CEST] [ozw.daemon] [info]: QT Version:  5.14.2 
1594392324: Client qt-openzwave-1 disconnected.
In exit
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
1594392324: mosquitto version 1.6.9 terminating
1594392324: Saving in-memory database to /data/mosquitto.db.
s6-svwait: fatal: supervisor died
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.```

### LINKED ISSUE #1462: OZW addon 0.5.1 does not startup
## The problem

OZW addon updated from 0.5.0 to 0.5.1. 0.5.1 does not not start up; see log. No Zwave communication is possible.
OZW addon 0.5.0 was working well.

## Environment

- Home Assistant Core release with the issue: 0.112.4
- Last working Home Assistant Core release (if known): unknown
- Operating environment (OS/Container/Supervised/Core): 4.11
- Integration causing this issue: OZW 0.5.1
- Link to integration documentation on our website:  https://www.home-assistant.io/integrations/ozw/

## Problem-relevant `configuration.yaml`

N/A
```yaml

```

## Traceback/Error logs

```txt
[20200714 14:58:19.185 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/isnodefailed/") 
[20200714 14:58:19.185 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "open" 
[20200714 14:58:19.186 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/open/") 
[20200714 14:58:19.186 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "ping" 
[20200714 14:58:19.190 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/ping/") 
[20200714 14:58:19.191 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "refreshnodeinfo" 
[20200714 14:58:19.191 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/refreshnodeinfo/") 
[20200714 14:58:19.191 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "refreshvalue" 
[20200714 14:58:19.191 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/refreshvalue/") 
[20200714 14:58:19.191 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "removeassociation" 
[20200714 14:58:19.191 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/removeassociation/") 
[20200714 14:58:19.192 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "removefailednode" 
[20200714 14:58:19.192 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/removefailednode/") 
[20200714 14:58:19.192 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "removenode" 
[20200714 14:58:19.192 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/removenode/") 
[20200714 14:58:19.192 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "replacefailednode" 
[20200714 14:58:19.192 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/replacefailednode/") 
[20200714 14:58:19.192 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "requestallconfigparam" 
[20200714 14:58:19.193 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/requestallconfigparam/") 
[20200714 14:58:19.193 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "requestconfigparam" 
[20200714 14:58:19.194 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/requestconfigparam/") 
[20200714 14:58:19.194 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "requestnetworkupdate" 
[20200714 14:58:19.195 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/requestnetworkupdate/") 
[20200714 14:58:19.195 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "requestnodedynamic" 
[20200714 14:58:19.195 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/requestnodedynamic/") 
[20200714 14:58:19.195 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "requestnodeneighborupdate" 
[20200714 14:58:19.198 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/requestnodeneighborupdate/") 
[20200714 14:58:19.199 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "requestnodestate" 
[20200714 14:58:19.200 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/requestnodestate/") 
[20200714 14:58:19.201 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "sendnodeinformation" 
[20200714 14:58:19.202 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/sendnodeinformation/") 
[20200714 14:58:19.203 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "setpollinterval" 
[20200714 14:58:19.204 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/setpollinterval/") 
[20200714 14:58:19.205 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "setvalue" 
[20200714 14:58:19.206 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/setvalue/") 
[20200714 14:58:19.206 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "softresetcontroller" 
[20200714 14:58:19.208 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/softresetcontroller/") 
[20200714 14:58:19.208 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "syncronisenodeneighbors" 
[20200714 14:58:19.209 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/syncronisenodeneighbors/") 
[20200714 14:58:19.209 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "testnetwork" 
[20200714 14:58:19.211 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/testnetwork/") 
[20200714 14:58:19.211 CEST] [ozw.mqtt.commands] [debug]: Creating Subscription for  "testnetworknode" 
[20200714 14:58:19.212 CEST] [ozw.mqtt.commands] [debug]: Subscription Setup for  QMqttTopicFilter("OpenZWave/1/command/testnetworknode/") 
[20200714 14:58:19.217 CEST] [ozw.mqtt.publisher] [debug]: Topics:  "OpenZWave/1/status/" 
[20200714 14:58:19.218 CEST] [ozw.mqtt.publisher] [warning]: Another ozwdaemon is running for Instance  1 
[20200714 14:58:19.219 CEST] [ozw.mqtt.publisher] [warning]: If not, please clean up the MQTT Topic:  "OpenZWave/1/status/" 
[20200714 14:58:19.220 CEST] [ozw.mqtt.publisher] [warning]: "{\n    \"OpenZWave_Version\": \"1.6.1198\",\n    \"OZWDaemon_Version\": \"0.1.0\",\n    \"QTOpenZWave_Version\": \"1.2.0\",\n    \"QT_Version\": \"5.12.5\",\n    \"Status\": \"driverRemoved\",\n    \"TimeStamp\": 1594709841,\n    \"homeID\": 0\n}" 
[20200714 14:58:19.222 CEST] [ozw.library] [debug]: Detail - Node: 0 Notification: DriverRemoved 
[20200714 14:58:19.223 CEST] [ozw.notifications] [debug]: Notification pvt_driverRemoved  0 Thread:  0x76f09558 
[20200714 14:58:19.224 CEST] [ozw.mqtt.publisher] [debug]: Publishing Event stopped 
[20200714 14:58:19.225 CEST] [ozw.mqtt.publisher] [debug]: Publishing Event driverRemoved: 0 
[20200714 14:58:19.227 CEST] [ozw.library] [debug]: Always - Node: 0 *************************************************************************** 
[20200714 14:58:19.228 CEST] [ozw.library] [debug]: Always - Node: 0 *********************  Cumulative Network Statistics  ********************* 
[20200714 14:58:19.229 CEST] [ozw.library] [debug]: Always - Node: 0 *** General 
[20200714 14:58:19.231 CEST] [ozw.library] [debug]: Always - Node: 0 Driver run time: . .  . 0 days, 0 hours, 0 minutes 
[20200714 14:58:19.232 CEST] [ozw.library] [debug]: Always - Node: 0 Frames processed: . . . . . . . . . . . . . . . . . . . . 0 
[20200714 14:58:19.233 CEST] [ozw.library] [debug]: Always - Node: 0 Total messages successfully received: . . . . . . . . . . 0 
[20200714 14:58:19.234 CEST] [ozw.library] [debug]: Always - Node: 0 Total Messages successfully sent: . . . . . . . . . . . . 0 
[20200714 14:58:19.236 CEST] [ozw.library] [debug]: Always - Node: 0 ACKs received from controller:  . . . . . . . . . . . . . 0 
[20200714 14:58:19.237 CEST] [ozw.library] [debug]: Always - Node: 0 *** Errors 
[20200714 14:58:19.238 CEST] [ozw.library] [debug]: Always - Node: 0 Unsolicited messages received while waiting for ACK:  . . 0 
[20200714 14:58:19.239 CEST] [ozw.library] [debug]: Always - Node: 0 Reads aborted due to timeouts:  . . . . . . . . . . . . . 0 
[20200714 14:58:19.241 CEST] [ozw.library] [debug]: Always - Node: 0 Bad checksum errors:  . . . . . . . . . . . . . . . . . . 0 
[20200714 14:58:19.241 CEST] [ozw.library] [debug]: Always - Node: 0 CANs received from controller:  . . . . . . . . . . . . . 0 
[20200714 14:58:19.243 CEST] [ozw.library] [debug]: Always - Node: 0 NAKs received from controller:  . . . . . . . . . . . . . 0 
[20200714 14:58:19.243 CEST] [ozw.library] [debug]: Always - Node: 0 Out of frame data flow errors:  . . . . . . . . . . . . . 0 
[20200714 14:58:19.245 CEST] [ozw.library] [debug]: Always - Node: 0 Messages retransmitted: . . . . . . . . . . . . . . . . . 0 
[20200714 14:58:19.246 CEST] [ozw.library] [debug]: Always - Node: 0 Messages dropped and not delivered: . . . . . . . . . . . 0 
[20200714 14:58:19.247 CEST] [ozw.library] [debug]: Always - Node: 0 *************************************************************************** 
[20200714 14:58:19.247 CEST] [ozw.library] [warning]: Warning - Node: 0 WARNING: Tried to write driver config with no home ID set 
[20200714 14:58:19.250 CEST] [ozw.library] [info]: Info - Node: 0 Stopping DNSThread 
ozwadmin: Settings Path:  "/root/.config/OpenZWave/ozw-admin.conf"
ozwadmin: Using Saved Network Cache Size: 1000
QT_MESSAGE_PATTERN: %{backtrace} is not supported by this Qt build
[20200714 14:58:19.549 CEST] [ozwadmin] [info]: OZWCore Initilized QTOpenZwave(0x757db0b0) QTOZWManager(0x19bd1a0) 
[20200714 14:58:19.639 CEST] [default] [warning]: QVncServer created on port 5900 
[20200714 14:58:21.308 CEST] [ozw.library] [debug]: Debug - Node: 0 Downloads Remaining: 0 
[20200714 14:58:21.308 CEST] [ozw.library] [info]: Info - Node: 0 ManufacturerSpecificDB Initialized 
WebSocket server settings:
  - Listen on :8099
  - Web server. Web root: /usr/share/novnc
  - No SSL/TLS support (no cert file)
  - proxying from :8099 to 127.0.0.1:5900
[20200714 14:58:23.311 CEST] [ozw.library] [info]: Info - Node: 0 mgr,     Driver for controller /dev/serial/by-id/usb-0658_0200-if00 removed 
[20200714 14:58:23.311 CEST] [ozw.manager] [debug]: Driver Removed for  "/dev/serial/by-id/usb-0658_0200-if00" 
[20200714 14:58:23.311 CEST] [ozw.manager] [debug]: OZW Serial Port Closed 
[20200714 14:58:23.311 CEST] [ozw.daemon] [info]: Shutting Down  "ozwdaemon"  Version:  "0.1.0" 
[20200714 14:58:23.312 CEST] [ozw.daemon] [info]: OpenZWave Version:  "1.6.1198" 
[20200714 14:58:23.312 CEST] [ozw.daemon] [info]: QT-OpenZWave Version:  "1.2.0" 
1594731503: Client qt-openzwave-1 disconnected.
[20200714 14:58:23.312 CEST] [ozw.daemon] [info]: QT Version:  5.12.5 
In exit
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
1594731503: mosquitto version 1.6.8 terminating
1594731503: Saving in-memory database to /data/mosquitto.db.
s6-svwait: fatal: supervisor died
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
```

## Additional information

using beta-channel

### PR DESCRIPTION
**Pull request by frenck**
## 0.5.2

- Fix startup failure due to stray OZW Daemon status retained in MQTT
- Propagate shutdown OZW Daemon status to upstream MQTT on shutdown
- Update OpenZWave to 6cf3729

fixes #1462
fixes #1454

### CODE REVIEW

File: zwave/rootfs/etc/cont-finish.d/mqtt.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    username=$(bashio::services "mqtt" "username")
+
+    if bashio::config.has_value 'instance'; then
+        ozw_instance=$(bashio::config 'instance')
+    else
+        ozw_instance=1
+    fi
+
+    # Ensure upstream MQTT server has the right OZW status on shutdown
+    # In this case, the LTW is no relayed to the upstream MQTT server
```
Comment (MartinHjelmare) [inline zwave/rootfs/etc/cont-finish.d/mqtt.sh:26]: In this case, the LWT is not relayed

File: zwave/rootfs/etc/cont-init.d/mqtt.sh [hunk 2]
```diff
@@ -33,5 +35,39 @@ else
...
+
+    if bashio::config.has_value 'instance'; then
+        ozw_instance=$(bashio::config 'instance')
+    else
+        ozw_instance=1
+    fi
+
+    # Ensure upstream MQTT server has the right OZW status
+    # Workaround for an incorrect retained OZW status in MQTT
+    # In this case, the LTW is no relayed to the upstream MQTT server.
```
Comment (MartinHjelmare) [inline zwave/rootfs/etc/cont-init.d/mqtt.sh:48]: As above.

File: zwave/rootfs/etc/services.d/zwave/run [hunk 3]
```diff
@@ -20,6 +20,33 @@ fi
...
+then
+    mosquitto_pub \
+        --host 127.0.0.1 \
+        --port 1883 \
+        --username ozw \
+        --pw "${MQTT_PASSWORD}" \
+        --retain \
+        --topic "OpenZWave/${OZW_INSTANCE}/status/" \
+        --message "$(bashio::var.json "Status" "Offline")"
+fi
```
Comment (pvizeli) [inline zwave/rootfs/etc/services.d/zwave/run:48]: I think it's 3 time the same function, can we offload it as function `/usr/libs/mqtt_helper.sh` and include the function on the scripts to call it?

Comment (frenck) [reply zwave/rootfs/etc/services.d/zwave/run:48]: I guess so.

Comment (frenck) [reply zwave/rootfs/etc/services.d/zwave/run]: Done 👍

**Comment by kpine** [comment]
Hi. This breaks the OZW integration. Latest OZW introduces an incompatibility with the existing API. The Dimming Duration value type changed from a byte encoded in z-wave values to a duration time in seconds.

https://github.com/OpenZWave/open-zwave/commit/f81bc04b672d52b7bd5f8e8b604058c170c9911b

This means when a user toggles a light switch, HA will try to encode the seconds value. The result is for the factory transition time (no transition time) is 255 seconds. Transition times > 127 are also affected.

https://github.com/home-assistant/core/blob/aa1c5fc43d78edf287440df3f5c2d0b5acb3b3cd/homeassistant/components/ozw/light.py#L155

**Comment by MartinHjelmare** [comment]
Thanks for the report. So we should fix this in the core repo. @kpine if you have time please open an issue there so we keep track of this. Copy the above report is good.

**Comment by kpine** [comment]
https://github.com/home-assistant/core/issues/38068

**Pull request merged**